src/Core/Form/IdentifiableObject/DataProvider/CategoryFormDataProvider.php

```diff
@@ -101,15 +102,11 @@ public function getData($categoryId)
      */
     public function getDefaultData()
     {
-        $defaultGroups = $this->defaultGroupsProvider->getGroups();
+        $allGroupIds = GroupDataProvider::getAllGroupIds();
```

Comment:
```suggestion
        $allGroupIds = $this->defaultGroupsProvider->getAllGroupIds();
```

Reply:
@Progi1984 getAllGroupIds is GroupDataProvider class and no more in DefaultGroupsProvider, how do I create an instance here ?
Why not in static ?

Reply:
@matks @Progi1984 I've try something without static and replacing DefaultGroupsProvider by GroupDataProvider in class param.
I added a new service in group.yml
I really don't know how to correctly add this single line of code in this mess. I hope you'll not talk about a bc break again.
Instead of telling me that I can't do that or that, I'd appreciate a help on how to do this correctly.

Reply:
@Progi1984 what do you think of this new version ?